### ISSUE TITLE
JAWS is reading the entire contents of the Package Details Tab when first visiting it

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#13878]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

17.13 Preview 1

### Impact

Other

### Repro Steps & Context

With the new tabs introduced to the Details Pane as part of the README work, there's an accessibility issue with JAWS that should be addressed.

JAWS is reading the entire contents of the Package Details pane when first visiting it. All labels are read for some reason (eg, "Description Version colon"). I've not seen this before so will take some investigation to understand what's happening.
Narrator did not do this, if I recall correctly.

![image](https://github.com/user-attachments/assets/f1bb522a-a12f-44ed-9a9d-6171d8c3fced)

**Comment by nkolev92** [comment]
@jgonz120 Assigning a couple of bugs related to the Readme work.

**Comment by nkolev92** [comment]
Fixed by https://github.com/NuGet/NuGet.Client/pull/6141
